src/olympia/lib/settings_base.py

```diff
@@ -1759,10 +1752,22 @@ def read_only_mode(env, retry_after=None):
     path('static'),
 )
 
-NETAPP_STORAGE = TMP_PATH
-GUARDED_ADDONS_PATH = os.path.join(ROOT, 'guarded-addons')
+# Path related settings. In dev/stage/prod `NETAPP_STORAGE_ROOT` environment
+# variable will be set and point to our NFS/EFS storage
+# Make sure to check overwrites in conftest.py if new settings are added
+# or changed.
+STORAGE_ROOT = env('NETAPP_STORAGE_ROOT', default=path('storage'))
+
+ADDONS_PATH = os.path.join(STORAGE_ROOT, 'files')
+GUARDED_ADDONS_PATH = os.path.join(STORAGE_ROOT, 'guarded-addons')
+GIT_FILE_STORAGE_PATH = os.path.join(STORAGE_ROOT, 'git-storage')
+
+SHARED_STORAGE = os.path.join(STORAGE_ROOT, 'shared_storage')
+
+MEDIA_ROOT = os.path.join(SHARED_STORAGE, 'uploads')
+TMP_PATH = os.path.join(SHARED_STORAGE, 'tmp')
 
-GIT_FILE_STORAGE_PATH = os.path.join(MEDIA_ROOT, 'git-storage')
+REVIEWER_ATTACHMENTS_PATH = os.path.join(MEDIA_ROOT, 'reviewer_attachment')
```

Comment:
fwiw, I didn't find reviewer_attachment used anywhere when I was working on the jinja helper removal.

Reply:
Indeed, it's only used in the monitoring but that's it. May be worth removing, but that's not necessarily part of this patch

Reply:
seems like the perfect time to remove an unused setting :)

Reply:
fair point… I'll just do it :man_shrugging: